lib/index.js

```diff
@@ -21,6 +21,10 @@ function electronPrompt(options, parentWindow) {
 				resizable: false,
 				title: 'Prompt',
 				label: 'Please input a value:',
+				buttonLabels: {
```

Comment:
Put null instead of an object with default values, those labels are already present in the html

Reply:
Good point, done..